src/InputStepper/helpers/validateDecrement.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+import type { ValidateDecrement } from "./validateDecrement.js.flow";
+
+const validateDecrement: ValidateDecrement = ({
+  value,
+  minValue = Number.NEGATIVE_INFINITY,
+  step = 1,
+}) => {
+  const newValue = value - step;
+  const calculatedValue = newValue <= +minValue ? minValue : newValue;
+  if (calculatedValue !== value) {
```

Comment:
I think that it would be better to just return `true` or `false`.
Example usage:
```jsx
const handleDecrement = ev => {
  if(validateDecrement(ev.target.value, minValue, step)) {
    this.setState({ value: `${ev.target.value} steps` })
  }
}
```

Reply:
Also, same for the `validateIncrement`.